gsadjust/GSadjust.py

```diff
@@ -1980,8 +2002,5 @@ def main():
     else:
         ex.close()
 
-# Import here to avoid circular import error
-from gui_objects import show_message, SelectAbsg, AdjustOptions, LoopOptions
```

Comment:
I moved these back to the top, as I don't see the circular import error (is it after packaging?)

Importing here *shouldn't* really help as they will still be loaded as this file is imported. To prevent circular imports you usually need to import within a function, although it's best to avoid this if possible.